resources/assets/js/components/JobBuilder/Review/JobReview.tsx

```diff
@@ -784,15 +786,20 @@ export const JobReview: React.FunctionComponent<JobReviewProps &
           />{" "}
           <span data-c-colour="c2">{localizeField(locale, job, "title")}</span>
         </h3>
-        <p>
+        <p data-c-margin="bottom(double)">
           <FormattedMessage
             id="jobBuilder.review.headsUp"
             defaultMessage="Just a heads up! We've rearranged some of your information to help you
             understand how an applicant will see it once published."
             description="Description under primary title of review section"
           />
         </p>
-        <JobReviewActivityFeed jobId={job.id} isHrAdvisor={false} />
+        <ActivityFeed
+          jobId={job.id}
+          isHrAdvisor={false}
+          generalLocation={LocationId.jobGeneric}
+          locationMessages={jobReviewLocations}
```

Comment:
Once ActivityFeed accepts a filterComments function, we should get this to filter out comments not associated with the job builder (eg Application comments).